design/nations/nations-mockup.json

```diff
@@ -5,35 +5,35 @@
 		   	"type": "nation",
 		   	"region": "uganda",
 		   	"admin_name": "ugandabell",
-		   	"name": "UgandaBeLL"
+		   	"nation_name": "UgandaBeLL"
```

Comment:
this looks nice for the format but old db seems to use "name" right now. @dogi  do you want to keep it old way or change to new structure?